src/libs/antares/inifile/include/antares/inifile/inifile.h

```diff
@@ -68,7 +66,7 @@ class IniFile final
     class Section final
     {
     public:
-        Section();
+        Section() = default;
         explicit Section(const AnyString& name);
         ~Section();
```

Comment:
`= default` ?

Reply:
`Section() = default;` : I simply removed it.

Reply:
It was not `default` previously